Deleted two UX positions for HUD

amandaschonfeld opened the pull request:
Already have enough applicants and need to process everyone before the deadline.

amandaschonfeld committed:
Deleted two UX positions for HUD

Already have enough applicants and need to process everyone before the deadline.

fureigh commented on pages/open-positions.md:
```diff
@@ -32,8 +32,7 @@ If you’re interested in these opportunities, please email a detailed resume to
 
-- [User Experience Designer, Information Architecture]({{ site.baseurl }}/raas/hud/user-experience-designer-information-architecture/)
-- [User Experience Designer, Research]({{ site.baseurl }}/raas/hud/user-experience-designer-research/)
+
```
An extra linebreak snuck in. Please delete it if you can. Thanks!

fureigh commented:
@amandaschonfeld I took a look and added one line-specific comment. Please also delete these positions' respective files in the `_positions` directory by navigating to them and clicking the trash icon, then adding each of those changes to this pull request. ccing @rooneywp in case of questions. Thanks!

Commit:
Delete hud-fellow-user-experience-designer-research.md

Commit:
Delete hud-fellow-user-experience-information-architecture.md

Commit:
Update open-positions.md

pixelwake commented:
@fureigh I deleted the files and committed the changes to this branch.

fureigh commented:
@rooneywp Looks good to me. Thanks!

fureigh committed:
Remove stray linebreak, update deadline tense

fureigh committed:
Remove priority consideration deadline

Otherwise potential applicants may mistakenly think it's too late to apply for the remaining roles.

fureigh commented:
Made that remaining change and removed the priority consideration deadline, as it's passed. Merging.
